internal/ethapi/api.go

```diff
@@ -2082,10 +2076,7 @@ func (s *DebugAPI) GetRawTransaction(ctx context.Context, hash common.Hash) (hex
 		if tx = s.b.GetPoolTransaction(hash); tx != nil {
 			return tx.MarshalBinary()
 		}
-		if err == nil {
-			return nil, nil
-		}
-		return nil, NewTxIndexingError()
+		return nil, err
```

Comment:
It's wrong. Originally if the transaction is not found and background indexing is not running, `null` is returned.

Now, we return whatever error we have to the RPC consumers. It's a behavioral change.